### ISSUE TITLE
Monochrome icons not diplayed anymore on Linux

### ISSUE THREAD

**Issue by mlaverdiere** [nextcloud/desktop#1019]
I'm on Linux Fedora 29, on Gnome Shell, with ND client 2.5.1 set up to use ND monochrome icons.
All was working well but since a couple of days, monochrome icons are not displayed anymore, they are replaced by small coloured icons, either with the appindicator (kstatusnotifier) extension or topicon (redux) extension.

**Comment by mlaverdiere** [comment]
Also reported on Fedora bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1666341

**Comment by mlaverdiere** [comment]
Could be a package bug, since the 2.5.1 version of the appimage provided by Nextcloud (https://nextcloud.com/install/#install-clients) does not have this problem.

**Comment by janvlug** [comment]
I have two Fedora 29 systems. For one monochrome icons are working, for the other not. Both systems are running Gnome on Wayland.

**Comment by vc7ee** [comment]
I'm on Fedora 30.
![image](https://user-images.githubusercontent.com/38327267/57179591-f822d100-6e6e-11e9-8367-de8a5801e6a7.png)
I get this with or without monochrome.
Running from terminal:
```
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
xkbcommon: ERROR: Key "<CAPS>" added to modifier map for multiple modifiers; Using Control, ignoring Lock
```
for me `export QT_QPA_PLATFORM=wayland && nextcloud` works.

**To implement the temporary fix,** copy `/usr/share/applications/nextcloud.desktop` to ~/.local/usr/share/applications and edit the file:
change `Exec=nextcloud` to `Exec=env QT_QPA_PLATFORM=wayland nextcloud`.

**Comment by mlaverdiere** [comment]
This appears to be fixed for me, on Fedora 30 with ND client 2.5.2git.
